walletnode/services/artworkregister/task.go

```diff
@@ -185,19 +185,27 @@ func (task *Task) run(ctx context.Context) error {
 	return nil
 }
 
-func (task *Task) waitTxidValid(ctx context.Context, txID string, expectedConfirms int64, timeout time.Duration) error {
+func (task *Task) waitTxidValid(ctx context.Context, txID string, expectedConfirms int64, timeout time.Duration, interval time.Duration) error {
+	checkTimeout := func(checked chan<- bool) {
+		time.Sleep(timeout)
+		checked <- true
+	}
+
+	timeoutCh := make(chan bool)
```

Comment:
Just two things:
1. `chan struct{}` is better.
2.  `close(checked)` is better in my opinion.

Reply:
Thank you! Changes made